tests/Redux/todoReducers.test.js

```diff
@@ -29,4 +29,47 @@ describe('todos reducer', () => {
       },
     ]);
   });
+
+  it('should delete the item from the list', () => {
+    expect(
+      todo(undefined, { type: 'DELETE_TODO', ID: 1 }).todos
+    ).to.deep.equal([
+      {
+        ID: 0,
```

Comment:
@berzentine I suggest you pass a predefined application state instead of undefined, and then check if the actions are having any effect on it.
eg.

``` Javascript
const todoState = {
  todos: [
{
  ID: 0,
  completed: true,
  TEXT: 'Some random text',
},
{
  ID: 1,
  completed: true,
  TEXT: 'Some random text',
},
{
  ID: 2,
  completed: true,
  TEXT: 'Some random text',
},
}
``` 

and then try to delete let say ID 2. and please do similarly for other test cases.

Put it in one line, make the test cases more complex.